opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -469,3 +440,34 @@ message HistogramDataPoint {
   // measurements that were used to form the data point
   repeated Exemplar exemplars = 8;
 }
+
+// A representation of an exemplar, which is a sample input measurement.
+// Exemplars also hold information about the environment when the measurement
+// was recorded, for example the span and trace ID of the active span when the
+// exemplar was recorded.
+message Exemplar {
+  // The set of labels that were dropped by the aggregator, but recorded
+  // alongside the original measurement. Only labels that were dropped by the
+  // aggregator should be included
+  repeated opentelemetry.proto.common.v1.StringKeyValue dropped_labels = 1;
+
+  // time_unix_nano is the exact time when this exemplar was recorded
+  //
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+  fixed64 time_unix_nano = 2;
+
+  // Numerical value of the measurement that was recorded. Only one of these
+  // two fields is used for the data, based on MeasurementValueType.
+  double double_value = 3;
```

Comment:
Similar to Tigran's question above, should we distinguish raw integer and raw double points?

Reply:
This was just moved from the top of the file. But I tend to agree with #172 proposal you had, and just ignore 8bytes in memory overhead in order to simplify and not duplicate all the fields.